sdk/samples/iot/sample_pnp_device_info_component.c

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+#include "iot_sample_common.h"
```

Comment:
Does the ordering of this include matter compared to the others? That is, does it have to be first? Is that why you have a separate group for it?

Reply:
I've seen in many other samples, this include file going first as a group, so this is matching. Plus, in my case, there seems to be a bug in VS formatter - it puts this file non-alphabetically when inside a group (between `s` and `s` - impossible).